src/main/java/com/capstone/exff/ExffApplication.java

```diff
@@ -25,6 +31,29 @@ public static void main(String[] args) {
 
         return registrationBean;
     }
+    @Bean
+    public FilterRegistrationBean corsFilter() {
+        UrlBasedCorsConfigurationSource source = new UrlBasedCorsConfigurationSource();
+        CorsConfiguration config = new CorsConfiguration();
+        config.setAllowCredentials(true);
+        config.addAllowedOrigin("http://localhost:9000");
```

Comment:
should be http://localhost:8080

Reply:
It is "http://localshost:9000" because we allowed origin domain of client